### PR TITLE
Gcp ci firebase

### PR DESCRIPTION
**Pull request by GhiyasKM**
### What do you want to achieve?
- Updated cloudbuild.yml file for test, release Cloud build pipeline in gcp (firebase) for dynamic coded variable
- Have updated firebase.json using dynamic configuration for site hosting for test and release.

### PR Checklist

- [ ] I have tested it locally and all functionalities are working fine.
- [ ] I have compared it with mocks and all design elements are the same.
- [ ] I have tested it in IE Browser.

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

File: .firebaserc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "projects": {
-    "default": "km-chat-widget"
+    "default": "km-dev-434106"
```
Comment (adarshmishra) [inline .firebaserc:3]: @GhiyasKM project name should be dynamic, should not be hardcoded.

File: amplify.yml [hunk 2]
```diff
@@ -1,42 +1,42 @@
...
-            commands:
-                - echo "Build env:" $BUILD_ENV
-                - npm i
-        build:
-            commands:
-                # Kommunicate Test
-                - if [ "${BUILD_ENV}" = "test" ]; then npm run build-test; fi
-                # Kommunicate Release
-                - if [ "${BUILD_ENV}" = "release" ]; then npm run build-release; fi
-                # Kommunicate Staging
```
Comment (adarshmishra) [inline amplify.yml:18]: Check formatting, there should not be any changes in this file

File: firebase.json [hunk 3]
```diff
@@ -1,64 +1,11 @@
...
     "ignore": [
       "firebase.json",
       "**/.*",
       "**/node_modules/**"
-    ],
-    "headers": [
-      {
-        "source": "**",
-        "headers": [
-          {
```
Comment (adarshmishra) [inline firebase.json:13]: Why these headers are removed? From where these headers will be added?

File: cloudbuild.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  # Step 3: Deploy to Firebase Hosting
+  - name: 'gcr.io/$PROJECT_ID/firebase'
+    args:
+      - 'deploy'
+      - '--only'
+      - 'hosting:km-dev-434106-9a3b8'
+      - '--token'
+      - '${_FIREBASE_TOKEN}'
+      - '--project'
+      - 'km-dev-434106'
```
Comment (adarshmishra) [inline cloudbuild.yaml:42]: Project name should come from codepipe line, should not be hard coded in code

Comment (GhiyasKM) [reply cloudbuild.yaml]: Improved, Could you please review now.

File: .firebaserc (hunk 1 above)
Comment (GhiyasKM) [reply .firebaserc]: could you please review now

File: amplify.yml (hunk 2 above)
Comment (GhiyasKM) [reply amplify.yml:18]: Have Checked, Formatting is same

**Review by adarshmishra — APPROVED** [review]

**Pull request merged**